src/lib/electron.ts

```diff
@@ -498,4 +500,44 @@ export class SentryElectron implements Adapter {
       return originalEmit.call(emitter, event, ...args);
     };
   }
+
+  private normalizeEvent(event: any) {
+    const appPath = app.getAppPath();
+
+    if (event.culprit) {
+      event.culprit = this.normalizeUrl(event.culprit, appPath);
+    }
+
+    if (event.request && event.request.url) {
+      event.request.url =
+        this.normalizeUrl(event.request.url.replace('file:///', ''), appPath.replace(/\\/g, '/'));
+    }
+
+    const stacktrace =
+      event.stacktrace
```

Comment:
Is `event.stacktrace` to find the stacktrace in older browsers? Is this even required in Electron?